powerlevel9k.zsh-theme

```diff
@@ -547,6 +547,26 @@ prompt_history() {
   "$1_prompt_segment" "$0" "$2" "244" "$DEFAULT_COLOR" '%h'
 }
 
+# Detection for virtualization (systemd based systems only)
+prompt_detect_virt() {
+  if ! command -v systemd-detect-virt;then
+    return
+  fi
+  local virt=$(systemd-detect-virt)
+  local color="yellow"
+  if [[ "$virt" == "none" ]]; then
+    if [[ "$(ls -di / | grep -o 2)" != "2" ]]; then
+      virt="chroot"
+      "$1_prompt_segment" "$0" "$2" "$color" "$virt" ""
```

Comment:
`$1_prompt_segment` takes six argument, you just provide five. The first is the name of the function (`$0`). Second is the index (`$2`). Third is the background color (`$color`). Fourth is foreground color (`$virt` in your case; this is what looks odd to me).. After that the arguments seem shifted.